js/nodes/processes/constraints_process_list.js

```diff
@@ -0,0 +1,72 @@
+function ConstraintsProcessList () {
```

Comment:
I'd rename the class to `AssignVectorVariableProcess` as the Kratos base one. This would simplify a lot going through the code.

Reply:
Done